src/Bundle/Twig/TwigExtension.php

```diff
@@ -62,12 +62,20 @@ public function getFunctions()
      */
     public function webpackAsset($asset)
     {
-        $asset_id      = rtrim($this->public_path, '/') . '/' . Compiler::getAliasId($asset);
-        $document_root = isset($_SERVER['DOCUMENT_ROOT']) ? $_SERVER['DOCUMENT_ROOT'] : '';
+        // @FIXME: This method is used by the compiler which may or may not run from a CLI-environment. In this case the
+        //         variable DOCUMENT_ROOT isn't available and this method may produce unexpected behavior.
+
+        $asset_id        = rtrim($this->public_path, '/') . '/' . Compiler::getAliasId($asset);
+        $document_root   = isset($_SERVER['DOCUMENT_ROOT']) ? $_SERVER['DOCUMENT_ROOT'] : '';
+        $full_asset_path = $document_root . '/' . $asset_id;
```

Comment:
Making sure a compiled file exists. Otherwise you'd be spammed with 404-messages in the console in your browser :')